app/views/shared/_header.html.haml

```diff
@@ -25,7 +25,7 @@
               %li.navbar-text= "#{acting_user.full_name} (#{acting_user.username})"
               %li.divider-vertical
               = render "/shared/support"
-              %li= link_to "#{t('pages.cart')} (#{current_cart.order_details.count})", :cart
+              %li.js--cart_count{ data: { url: orders_cart_count_path, link: cart_url } }
```

Comment:
I'd vote for rendering the page with link that just says "Cart" without the count, then add the count later when we get the response via JS